src/configure/helper/redirectLinkHelper.ts

```diff
@@ -0,0 +1,38 @@
+import { RestClient } from "typed-rest-client";
+
+export enum ServiceFramework {
+    Moda,
+    Vssf
+}
+
+export class RedirectLinkHelper {
+
+    public repoAnalysisRedirectUrl: string  = "https://aka.ms/AA87j8k";
+    public repoAnalysisUrl: string = "";
```

Comment:
IMO it looks unnecessary to have these as public properties. These should be private and can be used as a cache (so that mulitple invocation of loadAll() does not make redirect calls again and again). The load.. functions should return a data type containing url and framework and the caller should use the return value directly instead of depending on these public properties.

Reply:
Infact it would have been okay to not store anything in this class and use this purely as helper to run some logic. If in future we need to make multiple invocations, then we would think about cache etc. otherwise we are unnecessarily storing things.

Reply:
Yes. If we need to cache we can implement this as a singleton.